spec/spec_helper.rb

```diff
@@ -29,7 +29,7 @@ class TestMailer < ActionMailer::Base; end
   config.include RSpec::Rails::ControllerExampleGroup, file_path: /controller(.)*_spec.rb$/
   config.mock_with :rspec
 
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
```

Comment:
@anaumov is this to fix the callbacks not working due to it waiting for a commit that never happens? I'd be worried that this will break specs for end-users as well.

Reply:
Yes, if it sets to `use_transactional_fixtures  = true` specs running with `rails  ~> 4.0` didn't commit transaction. No emails will be sent and two specs are failed. `use_transactional_fixtures  = false` works fine (the build is green). I read that it could slow down specs, but they are still pretty fast.